source/index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <!-- <meta charset="utf-8"> -->
+    <!-- <meta http-equiv="x-ua-compatible" content="ie=edge"> -->
+    <title>Becky Bosshart</title>
+    <meta name="description" content="The website of Rebecca Bosshart, author, journalist, teacher, redhead.">
+    <!-- <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-status-bar-style" content="black">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+
+    <link rel="canonical" href="https://beckybosshart.com/"> -->
+    <link rel="stylesheet" type="text/css" href="main.css">
+</head>
+
+<body>
+    <header>
+        <nav>
+            <a href="#">Download CV</a>
+            <a href="#">Email</a>
+            <a href="#">Phone</a>
+        </nav>
```

Comment:
I actually don't think these are `nav`. There really isn't any navigation … maybe [`menu` and `menuitem`](http://webdesign.tutsplus.com/tutorials/introducing-the-html5-menu-and-menuitem-elements--cms-22269) is appropriate here? What do you think?